edgelet/aziot-edged/src/image_gc.rs

```diff
@@ -0,0 +1,163 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+use edgelet_core::{Module, ModuleRuntime};
+use std::collections::HashMap;
+use std::fs;
+use std::io::Write;
+use std::time::{Duration, UNIX_EPOCH};
+
+use crate::error::Error as EdgedError;
+
+// TODO: fix file name with correct path
+const FILE_NAME: &str = "abc.txt";
+
+pub(crate) async fn run_until_shutdown(
+    settings: edgelet_settings::docker::Settings,
+    runtime: &edgelet_docker::DockerModuleRuntime<http_common::Connector>,
+) -> Result<edgelet_core::WatchdogAction, EdgedError> {
+    // Run the GC once a day while waiting for any running task
+    let gc_period = std::time::Duration::from_secs(86400);
+
+    let mut gc_timer = tokio::time::interval(gc_period);
+    gc_timer.set_missed_tick_behavior(tokio::time::MissedTickBehavior::Delay);
+
+    log::info!("Starting MIGC...");
```

Comment:
I know this is very early code, but I want to make sure the user-visible logs don't use internal acronyms in the final version.

Reply:
I actually didn't know MIGC was an _internal_ acronym.

I'll change it, but good call out.